build/build.ts

```diff
@@ -285,24 +283,15 @@ async function generateFrontendJs(): Promise<void> {
 
   await bundle.write({
     preferConst: true,
-    format: "esm",
-    file: outputFile
-  });
-
-  const webpackConf = {
-    mode: MODE,
-    entry: outputFile,
-    resolve: {
-      aliasFields: ["module"]
+    format: "umd",
```

Comment:
Imho rollup does similar jobs comparing with webpack, could we just remove webpack for the build?
and when debugging front end code, notice some source map flag missing so add it

Reply:
Last I checked Rollup doesn't have good support for loading JS dynamically so we decided to stick with Webpack for the client side code.